agent/pkg/tunnel/protocol/tcp/tcpproxy.go

```diff
@@ -148,9 +148,9 @@ func (tp *TCPProxyService) GetProxyStream(targetNodeName, targetIP string, targe
 
 	msg.Reset()
 	if err = streamReader.ReadMsg(msg); err != nil {
-		err = stream.Reset()
-		if err != nil {
-			return nil, fmt.Errorf("Stream between %s reset err: %v", targetNodeName, err)
+		error := stream.Reset()
+		if error != nil {
+			return nil, fmt.Errorf("Stream between %s reset err: %v", targetNodeName, error)
```

Comment:
use %w for error wrap would be better
```suggestion
			return nil, fmt.Errorf("Stream between %s reset err: %w", targetNodeName, error)
```